nengo/neurons.py

```diff
@@ -230,10 +233,27 @@ def step_math(self, dt, J, output):
             Input currents associated with each neuron.
         output : (n_neurons,) array_like
             Output activities associated with each neuron.
+        rng : `numpy.random.mtrand.RandomState`
+            Random number generator for stochastic neuron types.
+
+        Notes
+        -----
+
```

Comment:
Will this empty line affect sphinx? I usually see no empty line here but maybe it doesn't matter.

Reply:
I don't think it makes a difference. It looks fine in the rendered docs. But we could remove it for consistency I guess.